### PR TITLE
TIP-1022: return modified date as millis from epoch

### PR DESCRIPTION
**Pull request by wkoetke**
The api route listFolder now returns modified date as millis from epoch, so we need to convert it back to readable format in the dx cmd. The apiserver PR is: https://github.com/dnanexus/nucleus/pull/2644

### CODE REVIEW

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -2622,19 +2622,24 @@ def list_developers(args):
...
             input_params={"folder": args.folder, "recurse": args.recurse, "timeout": args.timeout})
-
         for r in results["results"]:
             if (args.csv == True):
+                dateStr = render_timestamp(r["modified"]) if r["modified"] != 0 else ''
                 print("{}{}{}{}{}".format(
-                    r["modified"], DELIMITER(","), r["size"], DELIMITER(","), r["path"]))
+                    dateStr, DELIMITER(","), r["size"], DELIMITER(","), r["path"]))
             else:
+                dateStr = render_timestamp(r["modified"]) if r["modified"] != 0 else '                   '
```
Comment (durae) [inline src/python/dxpy/scripts/dx.py]: Please remove some extra whitespaces:

```suggestion
                dateStr = render_timestamp(r["modified"]) if r["modified"] != 0 else ''
```

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -2622,19 +2622,24 @@ def list_developers(args):
...
+
 def list_database_files(args):
     try:
         results = dxpy.api.database_list_folder(
             args.database,
             input_params={"folder": args.folder, "recurse": args.recurse, "timeout": args.timeout})
-
         for r in results["results"]:
             if (args.csv == True):
+                dateStr = render_timestamp(r["modified"]) if r["modified"] != 0 else ''
```
Comment (durae) [inline src/python/dxpy/scripts/dx.py]: you can move this above condition as you need dateStr in both if and else. An I would use `date_str` as we are in python now :)

File: src/python/dxpy/scripts/dx.py (hunk 1 above)
Comment (wkoetke) [reply src/python/dxpy/scripts/dx.py]: Done

File: src/python/dxpy/scripts/dx.py (hunk 2 above)
Comment (wkoetke) [reply src/python/dxpy/scripts/dx.py]: Done

File: src/python/dxpy/scripts/dx.py [hunk 3]
```diff
@@ -2622,19 +2622,24 @@ def list_developers(args):
...
         results = dxpy.api.database_list_folder(
             args.database,
             input_params={"folder": args.folder, "recurse": args.recurse, "timeout": args.timeout})
-
         for r in results["results"]:
+            date_str = render_timestamp(r["modified"]) if r["modified"] != 0 else ''
             if (args.csv == True):
-                print("{}{}{}{}{}".format(
-                    r["modified"], DELIMITER(","), r["size"], DELIMITER(","), r["path"]))
+		print("{}{}{}{}{}".format(
```
Comment (durae) [inline src/python/dxpy/scripts/dx.py]: Does this need one more space? Otherwise LGTM.

Comment (durae) [reply src/python/dxpy/scripts/dx.py]: Ok, it doesn't. Just github playing with my eyes.

**Pull request merged**